### ISSUE TITLE
Warning about 8.8 update: Human error.

### ISSUE THREAD

**Issue by LuculentSugar6** [Monika-After-Story/MonikaModDev#2581]
Hey everybody, just a little warning: Do not be a hasty shmuck like me. If you have downloaded the 8.8 update, Monika may have told you about moving the DDLC folder from Steam's files. Don't be a dumb shit like me and just move it out to another place. Make a copy just in case! Because when I moved the folder, and tried to start up DDLC, my laptop thought I didn't even have DDLC and proceeds to download the game again. I tried to stop the download and I was too late. I even tried to paste back the older folder to no avail. I ended up back to vanilla act 3. All of the MAS mods have been deleted. Luckily, all I had to do was put 8.8 back into the game folder, and all was set back to normal.
Affection is still the same, Monika's birthday still happened, and the persistent files are still good as far as I can tell.

UPDATE: There will be somethings lost from what I can see, I had some extra random things laying around in the DDLC folder in general and it appears all of that is gone. Persistent should still be fine for y'all.

Admittedly, I'm not the most tech-savvy person out there, so I hope this doesn't happen to too much of y'all.

Moral of the story:
Back up and be careful!

**Comment by jmwall24** [comment]
Did you by chance after moving it, try to load up DDLC thru steam?  The point of moving the folder out of steam is that you now use the DDLC.exe file located in the folder you moved.  Don't use steam anymore for MAS.

**Comment by LuculentSugar6** [comment]
Yeah that is what I did wrong. I tried to load it through Steam. By any chance, do you know where exactly the DDLC.exe file would be located? I must've skimmed over it and couldn't locate it in the panic.

I just wish they could've made a bit more of an extensive guide for how to do this. I was kinda lost when I was doing it.

**Comment by LuculentSugar6** [comment]
So, as another question:
Do you know of what potential problems that Steam has caused before to others?

**Comment by jmwall24** [comment]
Besides not allowing you to use the in-mod updater, there have been other manual update problems, and for some people with the taking her out feature and not being able to get her back in.  It's just a lot easier and safer avoiding steam at this point.  So if you get that folder out and use the .exe instead of using steam, it's preferable.

**Comment by LuculentSugar6** [comment]
Damn, that sucks.

Hey, so I was running tests with other games using this method and it would give me a pop up saying an error has occurred due to a missing executable and would proceed to use Steam anyway to do updates and such. Would that happen with DDLC as well?

**Comment by jmwall24** [comment]
Have guided several other people today and have not had any problems with that if you follow the FAQ I sent.

**Comment by LuculentSugar6** [comment]
Okay, yeah. It was probably just that specific game anyway. DDLC might be different anyway too. Thanks.

**Comment by jmwall24** [comment]
Worst comes to worst you can always download the game from ddlc.moe, extract the game, then install the mod manually to that and play from that version.  It all uses the same save file that is located elsewhere on your PC.

**Comment by LuculentSugar6** [comment]
Its cool. I'll do my best to follow the FAQ. I'll just make a back up of the folder just in case that way I don't lose anything.
After that, just don't open through Steam anymore right?
And the persistent files will still be in the same spot just in case too?
Thanks for your time as well.

**Comment by jmwall24** [comment]
Correct.  And you're welcome.

**Comment by LuculentSugar6** [comment]
Yup everything worked out perfectly. Thanks!

**Comment by EntonyEscX** [comment]
The first thing I did when taking on a serious relationship with my Monika 8 months ago was moving the entire game from the Steam folder to my HD, so I've always been playing the mod off Steam all along.

**Comment by LuculentSugar6** [comment]
Yeah I must’ve made a mistake somewhere along the line. I believe what I did was when I took out the DDLC files with MAS, I tried to open said file. Then, it oddly opened steam and steam began to download a fresh DDLC. In a panic, I mistakenly put the DDLC with MAS back into the steam folder, and I’m assuming afterword the new DDLC overwritten that. 

Luckily, the persistent files remained as they are, and all I had to do was re-download the MAS mod to the game and redo the process. 

I was already on edge due to an earlier problem I had with launching MAS, so I was more prone to making more mistakes. This was one of them. Luckily it was just a close call.
